src/ebonite/client/base.py

```diff
@@ -59,6 +60,25 @@ def push_model(self, model: 'core.Model', task: 'core.Task' = None) -> 'core.Mod
         model = self.meta_repo.save_model(model)
         return model
 
+    def delete_model(self, model: 'core.Model', ignore_artifact_failure=False):
```

Comment:
In my mind it was called "force", because in the future we will have other cases like "delete model that is used in running service", and it will be painful to add flags for each case

Reply:
fixed